korean/coco/coco.md

```diff
@@ -1,6 +1,7 @@
 핫초코를 사러 나온 당신... 돈은 있나요? 핫초코를 어디서 살건가요? 
 
-[돈이 없으면 뭘 어쩌겠어? 다시 방으로 돌아간다](../marshmallow.md) 집에서 잠이나 자!
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.